### ISSUE TITLE
What to pass into the neural network at run time?

### ISSUE THREAD

**Issue by cfpark00** [cfpark00/Simple-Annotation-GUI#7]
1. learning rate
2. num_max_epoch
3. early stopping condition??

Anything else?
